modules/microshift-gitops-adding-apps.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// microshift_running_apps/microshift-gitops.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-gitops-adding-apps_{context}"]
+= Creating GitOps applications on {microshift-short}
+
+You can create a custom YAML configuration to deploy and manage applications in your {microshift-short} service.
+
+.Prerequisites 
+
+* You installed the GitOps ArgoCD package
```

Comment:
```suggestion
* `microshift-gitops` packages is installed and Argo CD pods are running. 
```
I'd add here a link to how to do this prerequisite step that is described here: https://github.com/openshift/openshift-docs/pull/72974/files?short_path=1b72eb8#diff-1b72eb821edc3eac7a2919a4cdabb885dbcb0b179acaccbe8c7966833aefab77